source/_posts/A-lightweight-tooltip-popup-for-Android.md

```diff
@@ -0,0 +1,147 @@
+---
+title: How to add a lightweight Tooltip popup in your Android Application
+date: 2020-09-23 13:40:48
+author: Diwakar Singh
+category: Android
+tags:
```

Comment:
NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION, 
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING

use some other keyword in place of which you used, Because everyone is using same keyword in the blog.